src/goto-programs/lazy_goto_functions_map.h

```diff
@@ -117,6 +117,25 @@ class lazy_goto_functions_mapt final
     ensure_function_loaded_internal(name);
   }
 
+  std::unordered_set<irep_idt, irep_id_hash> get_converted_functions() const
+  {
+    std::unordered_set<irep_idt, irep_id_hash> result;
+    for(const auto &id_and_body : goto_functions)
+      result.insert(id_and_body.first);
+    return result;
+  }
+
+  /// Returns the GOTO function body for function with name 'id', or throws if
+  /// that function has not been converted or does not exist.
+  /// Contrast `at(const irep_idt &)`, which converts functions on demand.
```

Comment:
Same problem as in the other PR: I don't understand this comment as the code seems to be doing exactly that.